src/qt/optionsmodel.cpp

```diff
@@ -83,6 +83,11 @@ void OptionsModel::Init(bool resetSettings)
         settings.setValue("fCoinControlFeatures", false);
     fCoinControlFeatures = settings.value("fCoinControlFeatures", false).toBool();
 
+    if (!settings.contains("fEnablePSBTControls")) {
+        settings.setValue("fEnablePSBTControls", false);
+    }
```

Comment:
While that may be technically correct, code written that way can be more confusing. Our style guide states that brackets must be included even for code like this.

Reply:
I might be wrong, and it might be a new change in writing style. But the code above it doesn't use brackets for one-liner **if-else**.

```
    if (!settings.contains("strThirdPartyTxUrls"))
        settings.setValue("strThirdPartyTxUrls", "");
    strThirdPartyTxUrls = settings.value("strThirdPartyTxUrls", "").toString();

    if (!settings.contains("fCoinControlFeatures"))
        settings.setValue("fCoinControlFeatures", false);
    fCoinControlFeatures = settings.value("fCoinControlFeatures", false).toBool();
```

Reply:
We are trying to change code style gradually. Existing code may not adhere to the style guide, but modifications to it and newly added code must.

Reply:
> We are trying to change code style gradually. Existing code may not adhere to the style guide, but modifications to it and newly added code must.

Understood.

> Also, personally, I hate the bracket-less style so I'm not going to do this even if the style guide allows it. There is no requirement that such code needs to be written this way.

I get it. Bracket-less style have a chance of inducing unnecessary errors in case of adding some line inside the conditional statement.